remove deprecated dropshadows

lokesh-sagi125 opened the pull request:
## Description
remove deprecated dropshadows

#### Issue addressed
 https://github.com/learningequality/kolibri-design-system/issues/725

Addresses #725 



## Changelog
<!-- [DO NOT REMOVE-USED BY GH ACTION] CHANGELOG START -->

<!--
  - Fill in the changelog item(s) below. If there are more groups of closely
    related changes, prepare more changelog items for each one of them.
    At a minimum, always separete non-breaking changes from breaking changes.
  - This needs to be pasted to CHANGELOG.md before merging a PR.
  - See changelog guidelines https://www.notion.so/learningequality/DRAFT-Changelog-Guidelines-106b6ebbdeda4ba5b3b3e7c490c5a4fe and existing
    items in CHANGELOG.md as examples
 -->

  - **Description:** removed the deprecated dropshadows
  - **Products impact:**  none
  - **Addresses:** #725 
  - **Components:** -
  - **Breaking:** no
  - **Impacts a11y:** no
  - **Guidance:**  - 

<!-- [DO NOT REMOVE-USED BY GH ACTION] CHANGELOG END -->

lokesh-sagi125 committed:
remove deprecated dropshadows

ozer550 reviewed (dismissed):
I just saw that in the issue description it is mentioned rather to comment these out and not remove them completely.

lokesh-sagi125 committed:
Revert "remove deprecated dropshadows"

This reverts commit 20bd55cb54535d2372efb8bbe229d2b0a2f4cec7.

lokesh-sagi125 committed:
comment instead of delete

lokesh-sagi125 commented:
oh , sorry about that @ozer550  i got confused and thought its the other way around 😅, just reverted the delete and commented the deprecated dropshadows:).

lokesh-sagi125 committed:
Revert "comment instead of delete"

This reverts commit 00b5e2adb2277720df1b64ec7091cabf655c8cc3.

lokesh-sagi125 committed:
Update definitions.scss

lokesh-sagi125 committed:
Update definitions.scss

AlexVelezLl reviewed (changes requested):
Thank you @lokesh-sagi125! These are the right changes for this, however, for now, we are checking if these 3 values ​​1dp, 2dp and 6dp also aligned correctly to Studio's needs, so I will block the merge of this PR for now until we have a final confirmation from our design team :hugs: thank you!

AlexVelezLl commented on lib/styles/definitions.scss:
```diff
@@ -67,91 +67,91 @@ $core-time: 0.15s;
-  box-shadow: 0 3px 5px -1px rgba(0, 0, 0, 0.2), 0 5px 8px 0 rgba(0, 0, 0, 0.14),
-    0 1px 14px 0 rgba(0, 0, 0, 0.12);
-}
+// %dropshadow-3dp {
```
Could we add here a comment indicating that uncommenting needs to be discussed with designers and should be aligned with the guidelines.

AlexVelezLl reviewed (commented):
Alright, it seems that these will work fine with Studio :tada:. Lets just add this comment that was mentioned in the issue,and after that, this should be good to go!

lokesh-sagi125 committed:
add comment .

adding a comment indicating uncommenting needs to be discussed with the designers first

AlexVelezLl commented on .gitignore:
```diff
@@ -22,3 +22,4 @@ docs/jsdocs.js
 .idea
 kolibri-design-system.iml
 docs/pages/playground.vue
+package.json
```
Lets remove this line, please, we need to track the changes of the package.json

AlexVelezLl commented on package.json:
```diff
@@ -83,5 +83,8 @@
     "extends browserslist-config-kolibri"
-  ]
+  ],
+  "volta": {
```
In KDS we still use node 10. Please, lets remove this addition

AlexVelezLl reviewed (changes requested):
Hey @lokesh-sagi125! Lets remove these extra additions, and after that we can proceed to merge the PR. Thank you!

lokesh-sagi125 committed:
Revert "add comment ."

This reverts commit 78ca7482323fbb5fbad92ff761fe197cc2d3efa5.

lokesh-sagi125 committed:
Update definitions.scss

lokesh-sagi125 commented:
@AlexVelezLl  just removed the additions

AlexVelezLl reviewed (approved):
Thank you @lokesh-sagi125! This looks good to me, merging :tada:.
